docs/media-queries.md

```diff
@@ -95,7 +95,7 @@ const mq = Object.keys(breakpoints).reduce(
       typeof breakpoints[label] === 'string' ? '' : 'px'
     accumulator[label] = cls =>
       css`
-        @media (${breakpoints[label] + suffix}) {
+        @media (min-width:${breakpoints[label] + suffix}) {
```

Comment:
Thanks for the PR! I rewrote this while changing the docs and should've tested it better, this change breaks for the string case though, could you change it to something like this?
```jsx
let prefix = typeof breakpoints[label] === 'string' ? '' : 'min-width:'
// in the media query interpolation
prefix + breakpoints[label] + suffix